// Copyright 2023 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";

package mediapipe;

import "mediapipe/framework/calculator.proto";

message KeypointsTo3DPoseShapeCalculatorOptions {
  extend mediapipe.CalculatorOptions {
    optional KeypointsTo3DPoseShapeCalculatorOptions ext = 258658798;
  }

  // A textform protobuf has all the parameters needed for the reconstruction.
  optional string config_filename = 1;
}
